lib/raygun.ts

```diff
@@ -140,14 +141,70 @@ class Raygun {
     this._tags = tags;
   }
 
+  transport() {
```

Comment:
Is there a specific interface/type that can be specified between the transport types to keep the externally accessible props consistent?

Reply:
Good catch, added an annotation :)